packages/labs/ssr-react/src/lib/node/wrap-jsx.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import type {jsx, jsxs} from 'react/jsx-runtime';
+import type {jsxDEV} from 'react/jsx-dev-runtime';
+import {createElement, type ReactNode, type ElementType} from 'react';
+import {renderCustomElement} from './render-custom-element.js';
+import {isCustomElement} from '../utils.js';
+
+export const wrapJsx = (originalJsx: typeof jsx, originalJsxs: typeof jsxs) =>
+  originalJsx.name !== 'litPatchedJsx'
+    ? function litPatchedJsx<P extends {children?: ReactNode}>(
```

Comment:
Like @AndrewJakubowicz 's comment above, you might need to make sure that this name isn't mangled. You could also detect this function instance by setting and checking a well-known property on it, or assigning it to a property on `globalThis`.

Reply:
I removed the checking by function name so it is no longer a problem if the name gets minified.